dll/win32/mshtml/olecmd.c

```diff
@@ -241,6 +241,13 @@ static HRESULT exec_print(HTMLDocument *This, DWORD nCmdexecopt, VARIANT *pvaIn,
         return S_OK;
     }
 
+#ifdef __REACTOS__
+    // returning here fixes CORE-16884. Maybe use this until printing works.
```

Comment:
This comment looks redundant: `ERR()` should be enough.

Reply:
@SergeGautherie sorry for abusing this text field, but why can't you be added as a reviewer explicitly? Sometimes I would love to add you. e.g. for https://github.com/reactos/reactos/pull/5067

Reply:
That would be more contributional than nitpicking here. What Doug provides here has been reviewed already, and is really good enough for a temporary safety-guard. Resolved

Reply:
> why can't you be added as a reviewer explicitly?

How would I know?

Reply:
@JoachimHenze People not part of "ReactOS Developers" group in github cannot be added, for whatever reason.